### DISCUSSION TITLE
Participant gets a black screen

### DISCUSSION THREAD

**Discussion by EnigmaRaptor** [mistweaverco/bananas#38] [Q&A]
I'm using Nobara Linux, and the joiner was using Windows 11.  After sending the link to the joiner, he was able to join the session, but it was just black.

**Comment by gorillamoe** [comment]
The OS shouldn't really matter that much. Can you open bananas from terminal and start a session and see if there are any errors popping up?

**Comment by gorillamoe** [comment]
Closing this because of lack of feedback. Feel free to re-open if necessary.
